### PR TITLE
Perf test: bind server to one NUMA node

### PR DESCRIPTION
**Pull request by akuzm**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by akuzm** [comment]
http://developer.amd.com/wordpress/media/2012/10/LibNUMA-WP-fv1.pdf

`numa_set_localalloc` influences kernel functions that allocate memory (e.g. mmap), apparently there is no LD_PRELOAD of different allocators or other such tricks.

File: docker/test/performance-comparison/perf.py.orig [hunk 1]
```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/python3
```
Comment (alexey-milovidov) [inline docker/test/performance-comparison/perf.py.orig:1]: Accidentially committed?

**Comment by akuzm** [comment]
Using `numactl` leads to mystery segfaults all over the place. I couldn't find out the reason, closing until better times.

**Comment by azat** [comment]
>Using numactl leads to mystery segfaults all over the place. I couldn't find out the reason, closing until better times.

This may be because of jemalloc issue, see #15035

**Pull request closed without merging**